### ISSUE TITLE
SentryBlock since today

### ISSUE THREAD

**Issue by leo-mck** [a-legotin/InstaSharper#94]
First, thank you for your amazing work on this library.

Since today, an app I had running cannot login. I updated the library with the latest nuget package and my code accordingly but still the error persist. When I try to run the sample/demo code with the `LogLevel.All`, I get:

```
Starting demo of InstaSharper project
Logging in as MY_USER
22:21:  Request: https://i.instagram.com/
22:21:  Response: GET https://www.instagram.com/
22:21:  Content:
22:21:  got html content!
22:21:  Response: GET https://www.instagram.com/
22:21:  Content:
22:21:  got html content!
22:21:  ----------------------------------------------------------------------------------------------------
22:21:  Request: POST https://i.instagram.com/api/v1/accounts/login/
22:21:  Headers:
22:21:  Accept-Language:["en-US"]
22:21:  X-IG-Capabilities:["3boBAA=="]
22:21:  X-IG-Connection-Type:["WIFI"]
22:21:  User-Agent:["Instagram","12.0.0.7.91","Android","(23/6.0.1; 640dpi; 1440x2560; samsung; SM-G935F; hero2lte; samsungexynos8890; en_NZ)"]
22:21:  Properties:
{
  "X-Google-AD-ID": "847d5b98-f1cc-496a-80d1-39c4c23ef450",
  "signed_body": "31f95b19670c5f00a7201a8d53a4a4a37d966a1c7d3a78493159258968c704dd.{\"phone_id\":\"28484284-e646-4a29-88fc-76c2666d5ab3\",\"username\":\"MY_USER\",\"guid\":\"7f585e77-becf-4137-bf1f-84ab72e35eb4\",\"device_id\":\"android-61521af75c71ecb7\",\"password\":\"_MY_PASS\",\"login_attempt_count\":\"0\"}",
  "ig_sig_key_version": "4"
}
22:21:  Response: POST https://i.instagram.com/api/v1/accounts/login/
22:21:  Content:
22:21:  {"message": "Sorry, there was a problem with your request.", "status": "fail", "error_type": "sentry_block"}
```

I confirmed that I can still login on instagram with this user/password. I created a new account and tried with it and got the same error. Please let me know if I can help with any other information.

**Comment by leo-mck** [comment]
I changed my ip address and blocked my account. Now waiting it to be disabled and will report as soon as I can :)

**Comment by a-legotin** [comment]
Hi @leo-mck, yep, you need to follow [steps](https://github.com/a-legotin/InstaSharper/wiki/FAQ#how-to-unblock-account-from-sentry-block), unblock account and see. Anyway I gonna change APP key to newest soon.

**Comment by leo-mck** [comment]
I tested it again after blocking my account and when I tried, got the same sentry block message. The android app and the web interface works fine.

**Comment by a-legotin** [comment]
fixed in 1.3.8, and do not use custom request message without understanding how it works

**Comment by leo-mck** [comment]
Thanks, it is working perfectly now! I do not now how custom request messages work, the code is from the demo project on the repository :)
